nixos/modules/services/databases/couchdb.nix

```diff
@@ -130,7 +139,7 @@ in {
 
       configFile = mkOption {
         type = types.string;
-        default = "/var/lib/couchdb/couchdb.ini";
+        default = if useVersion2 then "/var/lib/couchdb/local.ini" else "/var/lib/couchdb/couchdb.ini";
```

Comment:
I wanted to use that, but in this case the default depends on another config option. Is that possible?